### PR TITLE
Use the return value from populate_data

### PR DESCRIPTION
**Pull request by zacharylawrence**

### CODE REVIEW

File: recidiviz/ingest/base_scraper.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ def _generic_scrape(self, params):
             # If we want to scrape data, we should either create an ingest_info
             # object or get the one that already exists.
             ingest_info = params.get('ingest_info', IngestInfo())
```
Comment (colincadams) [inline recidiviz/ingest/base_scraper.py]: I still think this line should go away, but that can be a separate PR 😃

Comment (zacharylawrence) [reply recidiviz/ingest/base_scraper.py]: I agree and also wanted to seperate it in a different PR
